chia/util/keyring_wrapper.py

```diff
@@ -506,11 +504,9 @@ def migrate_legacy_keyring_interactive(self):
         """
         from chia.cmds.passphrase_funcs import async_update_daemon_migration_completed_if_running
 
-        # Make sure the user is ready to begin migration.
-        response = self.confirm_migration()
-        if not response:
-            print("Skipping migration. Unable to proceed")
-            exit(0)
+        # Let the user know about the migration.
+        if not self.confirm_migration():
+            exit("Migration aborted, can't continue running any chia command.")
```

Comment:
maybe this can be changed to `"Migration aborted, can't run any chia commands."`